dashboard/config/routes.rb

```diff
@@ -395,6 +397,10 @@ module OPS
       # Routes used by UI test status pages
       get 'test_logs/*prefix/since/:time', to: 'test_logs#get_logs_since', defaults: { format: 'json' }
       get 'test_logs/*prefix/:name', to: 'test_logs#get_log_details', defaults: { format: 'json' }
+
+      namespace :projects do
+        get 'section/:section_id', to: 'section_projects#index', defaults: { format: 'json' }
+      end
```

Comment:
:( Outside the scope of this PR, but I wonder why this `api/v1` namespace is defined in multiple places. We should consolidate.

[line 330](https://github.com/code-dot-org/code-dot-org/blob/c2d98235695272ff5d389afa93209207c01faaa3/dashboard/config/routes.rb#L330-L334)
and [line 389](https://github.com/code-dot-org/code-dot-org/blob/c2d98235695272ff5d389afa93209207c01faaa3/dashboard/config/routes.rb#L389-L399)

I won't mind if you do this as part of this PR ;), but no need.